grafana/config/grafana.ini

```diff
@@ -63,8 +63,8 @@ http_port = {{cfg.listening_port}}
 ;enable_gzip = false
 
 # https certs & key file
-;cert_file =
-;cert_key =
+cert_file = {{cfg.cert_file}}
```

Comment:
I'd recommend wrapping this in an `if` to avoid outputting the `cert_file` and `cert_key` directives if no cert is provided.

Reply:
Thanks for the suggestion. I added it in. If you see a better way I am all for it. @predominant